cmd/manager/main.go

```diff
@@ -185,7 +185,7 @@ func main() {
 	}
 	httpClient.HTTPClient.Transport = transport
 
-	httpClient.RetryMax = 3
+	httpClient.RetryMax = 10
```

Comment:
Do you foresee any issues w/ 2sec * 10x delay rather than the previous 3 sec total?  I agree this allows more time (for pods to come up etc..) to get an HTTP response, but is anything else sensitive to this timing?